cypress/e2e/awx/administration/notifiers/notifiersListView.cy.ts

```diff
@@ -83,37 +80,71 @@ describe('Notifications: List View', () => {
     //Assert the behavior in the UI following the test action
   });
 
-  it.skip('can copy a Notification and assert that the copy action completed successfully', () => {
+  it('can copy a Notification and assert that the copy action completed successfully', () => {
     //Utilize a notification of any type created in the beforeEach hook
     //Assert the existence of the notification before copy
     //Assert the copy action
     //Assert the existence of the copied notification as well as the original
-    cy.navigateTo('awx', 'notification-templates');
```

Comment:
this line needs to be added back. once the login happens, our next action is navigating to notifiers